src/ioctls/vcpu.rs

```diff
@@ -1210,16 +1214,27 @@ impl VcpuFd {
     /// let vm = kvm.create_vm().unwrap();
     /// let vcpu = vm.create_vcpu(0).unwrap();
     ///
-    /// // KVM_GET_REG_LIST demands that the vcpus be initalized.
-    /// let mut kvi: kvm_bindings::kvm_vcpu_init = kvm_bindings::kvm_vcpu_init::default();
-    /// vm.get_preferred_target(&mut kvi).unwrap();
-    /// vcpu.vcpu_init(&kvi).expect("Cannot initialize vcpu");
+    /// // KVM_GET_REG_LIST on Aarch64 demands that the vcpus be initalized.
+    /// #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
+    /// {
+    ///     let mut kvi: kvm_bindings::kvm_vcpu_init = kvm_bindings::kvm_vcpu_init::default();
```

Comment:
nit:

```suggestion
    ///     let mut kvi = kvm_vcpu_init::default();
```

and add an import for `kvm_bindings::kvm_vcpu_init`